tasks/extract_filepaths_for_granule/test/unit_tests/testevents/task_event.json

```diff
@@ -56,13 +56,14 @@
         "private-bucket": "sndbx-cumulus-private",
         "public-bucket": "sndbx-cumulus-public",
         "file-buckets": 
-          [{"regex": ".*.h5$", "sampleFileName": "L0A_HR_RAW_product_0010-of-0420.h5", "bucket": "protected"},
-              {"regex": ".*.cmr.xml$", "sampleFileName": "L0A_HR_RAW_product_0010-of-0420.iso.xml", "bucket": "protected"},
-              {"regex": ".*.h5.mp$", "sampleFileName": "L0A_HR_RAW_product_0001-of-0019.h5.mp", "bucket": "public"},
-              {"regex": ".*.cmr.json$", "sampleFileName": "L0A_HR_RAW_product_0001-of-0019.cmr.json", "bucket": "public"
+            [
+              {"regex": ".*.h5$", "sampleFileName": "L0A_HR_RAW_product_0010-of-0420.h5", "bucket": {"name": "protected"}},
```

Comment:
Incorrect json schema here for this array. The value of bucket should be a string not an object. This points to a key in the buckets configuration.

Reply:
reverted back to format `{"regex": ".*.h5.mp$", "sampleFileName": "L0A_HR_RAW_product_0001-of-0019.h5.mp", "bucket": "public"},`